release.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+
+# release all packages
+
+cd packages
+
+for package in "core" "utilities" "cli" "validation" 
+do
+  (cd $package; npm run release)
```

Comment:
I'm not sure that `npm run release` is clearer than `npm publish` (to which all `release` scripts are equivalent).

Reply:
Think you are right.

Reply:
changed